diff --git a/node_modules/nextra-theme-docs/dist/layout.js b/node_modules/nextra-theme-docs/dist/layout.js
index 57a511f..8dc617c 100644
--- a/node_modules/nextra-theme-docs/dist/layout.js
+++ b/node_modules/nextra-theme-docs/dist/layout.js
@@ -23,16 +23,20 @@ const Layout = (t0) => {
     themeConfig = $[2];
   }
   let data;
-  if ($[3] !== themeConfig) {
+  if ($[3] !== themeConfig || $[24] !== children) {
     const {
       data: t12,
       error
-    } = LayoutPropsSchema.safeParse(themeConfig);
+    } = LayoutPropsSchema.safeParse({
+      ...themeConfig,
+      children
+    });
     data = t12;
     if (error) {
       throw z.prettifyError(error);
     }
     $[3] = themeConfig;
+    $[24] = children;
     $[4] = data;
   } else {
     data = $[4];
